### PR TITLE
Add example to test the output

### PR DESCRIPTION
**Pull request by rnestler**
The example should contain all possible markdown syntax elements

### CODE REVIEW

File: Example.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+ 1. item 1
+ 1. item 2
+ 1. item 3
+
+> Quotes should also be supported.
+
+What should we do with [links](https://github.com/dbrgn/mdr) and images ![image](https://upload.wikimedia.org/wikipedia/commons/thumb/4/48/Markdown-mark.svg/312px-Markdown-mark.svg.png "Markdown logo")?
+
+Code blocks should get fancy syntax highlighting:
+```Rust
```
Comment (dbrgn) [inline Example.md]: This is not valid markdown though. Only the Github flavored style.

We should still support it, as long as the Hoedown library can parse it.

Comment (dbrgn) [inline Example.md]: Also, I don't think this works without an empty line between the text and the code block.

Comment (rnestler) [inline Example.md]: So the syntax highlighting is only available in Github flavored style? Then if we want syntax highlighting we need to support it.

Comment (dbrgn) [inline Example.md]: Hoedown passes in a "language" parameter, so there must be some kind of support for language declaration or detection. I haven't figured out how to use it yet though.

**Comment by dbrgn** [comment]
Can you rename to `example.md`, add the empty line and squash? :) Thx!

**Comment by rnestler** [comment]
It does actually work currently without the empty line in front of the code block.

**Comment by dbrgn** [comment]
Ah, OK :)

**Comment by rnestler** [comment]
Ahem, that was a bit fast... I accidentally changed some whitespace in main.rs

**Comment by dbrgn** [comment]
Haha, didn't see that. No problem.

**Pull request merged**